drivers/filesystems/ext2/src/fsctl.c

```diff
@@ -1799,7 +1799,7 @@ Ext2DeleteReparsePoint (IN PEXT2_IRP_CONTEXT IrpContext)
 
     BOOLEAN             FcbLockAcquired = FALSE;
     BOOLEAN             MainResourceAcquired = FALSE;
-
+    
```

Comment:
Technically correct maybe, but I wouldn't do that. Open a PR in the source repo and remove it instead.

Reply:
Indeed

Reply:
removed